src/extension.ts

```diff
@@ -74,6 +76,11 @@ export function activate(context: ExtensionContext) {
         statusBar.text = `Coverage: ${lines.hit}/${lines.found} lines`;
         statusBar.show();
       }
+    } else {
+      const activeTextEditor = window.activeTextEditor;
+      if (activeTextEditor && (activeTextEditor.document.uri.fsPath.toLowerCase() === file)) {

```

Comment:
If it is possible for the `activeTextEditor.document.uri.fsPath` to not equal the `file`, then maybe we shouldn't pass file in and maybe should just use `activeTextEditor.document.uri.fsPath`?  Also, seems like we should do this for entire function.

Reply:
ok, refactored..

Should we also rewrite `onDidCloseTextDocument` to juse call `showStatus` and move `statusBar.hide()` into there?